### PR TITLE
Help

### PR DESCRIPTION
**Pull request by jreidinger**
## Problem

double tab on CLI for partitioner opens a module.

- https://bugzilla.suse.com/show_bug.cgi?id=1172340
- https://trello.com/c/8G4FikEf/1988-ostumbleweed-p5-1172340-several-yast-modules-can-be-started-by-typing-yast2-module-tabtab

## Solution

Print properly that CLI is not supported

## Testing

- *Tested manually*

## Screenshots

*If the fix affects the UI attach some screenshots here.*

### CODE REVIEW

File: src/lib/y2partitioner/cli.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2014 SUSE LLC.
```
Comment (ancorgs) [inline src/lib/y2partitioner/cli.rb:1]: 2014?

File: src/lib/y2partitioner/cli.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+module Y2Partitioner
+  # CLI support for storage-ng. It just say that there is no one.
+  class CLI
+    extend Yast::I18n
+
+    def self.run
+      cmdline_description = {
+        "id"   => "partitioner"
+      }
```
Comment (ancorgs) [inline src/lib/y2partitioner/cli.rb:29]: Does this actually print something?

Comment (jreidinger) [reply src/lib/y2partitioner/cli.rb:29]: yes, it prints all that is needed:
```
localhost:/home/jreidinger # yast2 partitioner help
YaST2 partitioner
-----------------

This YaST2 module does not support the command line interface.
```

for others I use help to suggest CLI tool, but for all partitioner functionality it is hard.

**Review by ancorgs — APPROVED** [review]
LGTM

**Review by ancorgs — APPROVED** [review]
Still LGTM after rubocop changes

**Pull request merged**